sys/callout.c

```diff
@@ -62,16 +62,17 @@ void callout_stop(callout_t *handle) {
  * position.
 */
 void callout_process(realtime_t time) {
-  int now = time % CALLOUT_BUCKETS;
-  bool done = false;
-
-  while (!done) {
-    int last = ci.last++ % CALLOUT_BUCKETS;
-
-    if (last == now)
-      done = true;
-
-    callout_head_t *head = &ci.heads[last];
+  unsigned int current_bucket = ci.last % CALLOUT_BUCKETS, last_bucket;
```

Comment:
`, last_bucket` confused me a bit. Please try not to not mix declaration and definition on a single line.